Models/DTOs/UserProfile/UserProfileForUserProfileDetails.cs

```diff
@@ -0,0 +1,50 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.AspNetCore.Identity;
+
+namespace Tabloid.Models.DTOs;
+public class UserProfileForUserProfileDetailsDTO
+{
+    public int Id { get; set; }
+
+    [Required]
+    [MaxLength(50)]
+    public string FirstName { get; set; }
+
+    [Required]
+    [MaxLength(50)]
+    public string LastName { get; set; }
+
+    [NotMapped]
+    public string UserName { get; set; }
+
+    [NotMapped]
+    public string Email { get; set; }
+
+    public DateTime CreateDateTime { get; set; }
+
+    [DataType(DataType.Url)]
+    [MaxLength(255)]
+    public string ImageLocation { get; set; }
+
+    [NotMapped]
+    public List<string> Roles { get; set; }
+
+    public string IdentityUserId { get; set; }
+
+    public IdentityUser IdentityUser { get; set; }
+
+    public string FullName
+    {
+        get
+        {
+            return $"{FirstName} {LastName}";
+        }
+    }
+
+    public bool IsActive { get; set; }
+    public string CreatedOnDate
```

Comment:
Could be made into a single line like this
```cs
public string CreatedOnDate => CreateDateTime.ToString("MM-dd-yyyy")
```

Reply:
Might also consider changing the property name to `FormattedCreateDateTime` to be in line with the name of the original property